src/main/java/org/commcare/resources/model/ResourceTable.java

```diff
@@ -1175,4 +1198,12 @@ public void setMissingResources(SizeBoundUniqueVector<Resource> missingResources
     public SizeBoundUniqueVector<Resource> getMissingResources() {
         return mMissingResources;
     }
+
+    public Vector<Resource> getLazyResources() {
+        return storage.getRecordsForValues(new String[]{Resource.META_INDEX_LAZY}, new Boolean[]{true});
```

Comment:
do we use this now that we're mostly iterating on ID's? This feels a bit like a potential bomb call, it'll grab a LOT of records.